lib/src/tag/tagger.dart

```diff
@@ -378,8 +378,10 @@ class Tagger {
             ? 'darwin'
             : darwinOs;
 
+        final podspecFile = path.join(osDir, '$packageName.podspec');
```

Comment:
where is the osDir coming from?

Reply:
So if a platform is using `sharedDarwinSource` we look in `darwin`, otherwise we look in the platform's name (eg. `ios`).

Reply:
Ah, sorry, I didn't do `git pull upstream...` before checking the surrounding code.